debian/mariadb-server-10.5.postinst

```diff
@@ -169,12 +169,6 @@ EOF
       fi
     fi
 
-    # copy out any mysqld_safe settings
```

Comment:
Typo in commit message: "in **its** current **form**"

Reply:
Fixed